utils/page_objects/base_page/base_page.js

```diff
@@ -0,0 +1,74 @@
+const { Builder, wait, until, By } = require("selenium-webdriver");
+const { driver } = require("../../webdriver/chrome_driver");
+const { Header } = require("./header");
+
+class BasePage {
+    constructor() {
+        this.baseUrl = `https://www.ah.nl`;
+        this.Header = new Header();
+    }
+
+    getPageUrl() {
+        return `${this.baseUrl}${this.path}`;
+    }
+
+    open() {
+        return driver.get(`${this.baseUrl}${this.path}`);
+    }
+
+    getTitle() {
+        return driver.getTitle();
+    }
+
+    getCurrentUrl() {
+        return driver.getCurrentUrl();
+    }
+
+    input(element, inputData) {
```

Comment:
Please use a more specific name here: sendKeys() for example. And again: don't break the single responsibility principle here, it should be a separate helper function.